deployment/helm-k8s/values.yaml

```diff
@@ -118,22 +125,23 @@ ui:
 
 manager:
   appName: "storm-topology-manager"
-  serviceAccount: "storm-topology-manager"
+  serviceAccountName: "storm-topology-controller"
```

Comment:
is it a reason to use different name? controller?

Reply:
that's the name we use in GR, the name can be anything so thought we could make it easier for us (one less value to change)

Reply:
OK thanks